src/main/resources/assets/nochatreports/lang/en_us.json

```diff
@@ -8,7 +8,7 @@
 	"gui.nochatreports.status_insecure": "This server demands all chat messages to be signed and you agreed to proceed with this arrangement. Everything you send in the chat is potentially incriminating evidence that can be used against you in the report. You can remove this server from §oWhitelisted Servers§r in the mod config to get warned again.",
 	"gui.nochatreports.status_realms": "This is a Realms server. Not only there is no way to prevent chat reports here, but Mojang also employ automated chat monitoring to identify potential \"online harms\" and escalate them for human review, whether owner of the realm likes it or not. Your only hope is chat encryption.",
 	"gui.nochatreports.status_insecure_whitelist_all_servers": "This server demands all chat messages to be signed and you disabled the full-screen warning for it. Everything you send in the chat is potentially incriminating evidence that can be used against you in the report. You can disable §oWhitelist all servers§r in the mod config to get warned again.",
-	"gui.nochatreports.status_unknown": "No Chat Reports could not evaluate the safety of this server. This should never happen, please report it as a bug to the mod developer.",
+	"gui.nochatreports.status_unknown": "No Chat Reports could not evaluate the safety of this server. This may happen if you toggle NCR after joining a server, in that case simply relog to get the correct state. If it happened in some other case, please report it as a bug to the mod developer.",
```

Comment:
> Unknown status can now happen more frequently due to the mod toggle in NCR settings, this is now clarified in the tooltip

It shouldn't, since server safety status is simply not displayed if mod toggle is turned off. If somehow it was happening in your instance then report it as a bug.

Reply:
The point is when you re-enable the mod... but ok, I'll report it and revert this string.